jukebox_config.ini

```diff
@@ -6,3 +6,10 @@ minLogLevel=INFO
 
 [RestAPI]
 port=8888
+
+[Spotify]
+port=8889
+clientID=f589b31542ca45a98c076460a021e086
+clientSecret=0a82fbaa5fae43f0904776d6f695a7ad
+redirectUri=http://localhost:8889/spotifyCallback
+scopes=user-read-private user-read-email app-remote-control user-modify-playback-state
```

Comment:
i needed the scope `user-read-playback-state` too. i wonder why it worked for you without it :thinking:

Reply:
i fetched the token always with the developer api .. (it was faster)